.github/workflows/ci.yml

```diff
@@ -79,6 +79,11 @@ jobs:
         run: noirup ${{steps.parse.outputs.toolchain}}
       - name: Check nargo installation
         run: nargo --version
+      - name: Check noir-profiler installation
+        run: |
+          if [ -f "${HOME}/.nargo/bin/noir-profiler" ]; then
+            noir-profiler --version
+          fi
```

Comment:
This test is a little tuatological. We're running `noir-profiler --version` to check if it's there but only if it's there.

What I've done elsewhere in this workflow is to split the version number into major, minor, patch and we can then assert that after a certain version we expect the profiler to exist.